imports/plugins/core/accounts/client/components/updateEmail.js

```diff
@@ -50,7 +50,7 @@ class UpdateEmail extends Component {
         />
         <Components.Button
           bezelStyle={"solid"}
-          icon={showSpinner && "fa fa-spin fa-circle-o-notch"}
+          icon={showSpinner ? "fa fa-spin fa-circle-o-notch" : ""}
```

Comment:
Style preference is the original `&&` versus ternary operators... (which aren't so cool, and generally dislike ternary operators because it results in hard to read code...). Let's revert this (and it has nothing to do with this PR).

Reply:
@spencern @aaronjudd  I'd made this fix here because this throws this console error

`Failed prop type: Invalid prop 'icon' of type 'boolean' supplied to 'Reaction(Button)', expected 'string'.`

 if `showSpinner` is false. So it was passing an empty string to the `icon` prop instead.

I'll, however, revert this and probably open a separate issue for it as it's not related to this PR.